packages/ember-data/lib/system/record-array-manager.js

```diff
@@ -27,6 +27,12 @@ export default Ember.Object.extend({
       defaultValue: function() { return []; }
     });
 
+    this.liveRecordArrays = MapWithDefault.create({
+      defaultValue: (typeClass) => {
+        return this.createRecordArray(typeClass);
```

Comment:
Unsure how this would be achieved since `filteredRecordArrays` has an array as value ...